src/ProjectTemplates/test/Helpers/Project.cs

```diff
@@ -196,6 +197,7 @@ internal AspNetProcess StartPublishedProjectAsync()
             var environment = new Dictionary<string, string>
             {
                 ["ASPNETCORE_URLS"] = _urls,
+                ["ASPNETCORE_Kestrel__EndpointDefaults__Protocols"] = "Http1"
```

Comment:
Oh?

Reply:
To workaround the ongoing HTTP/2.0 bug

Reply:
The Kestrel one? That's been reverted.

Reply:
I'll file a bug and do cleanup afterwards. We don't want to delay this fix.

Reply:
Well, I won't file a bug because I'll just make a PR right after I merge this. I don't want to do anything that destabilizes these tests at this point.